src/Tasks/Microsoft.NET.Build.Tasks/PreprocessPackageDependenciesDesignTime.cs

```diff
@@ -182,6 +191,82 @@ private void PopulateAssemblies()
             }
         }
 
+        /// <summary>
+        /// Update FileDefinitions and FileDependencies to pretend that certain Reference items
+        /// explicitly added by a .targets file in the NETStandard.Library are actually normal
+        /// package items. This allows them to show up properly under the "SDK" node in Solution
+        /// Explorer, rather than just under the "Assemblies" node.
+        ///
+        /// This is not meant to be a general mechanism for injecting files that can't be handled
+        /// through the normal NuGet package mechanisms. For that, package authors should be
+        /// creating FileDefinition and FileDependency items instead of Reference items.
```

Comment:
@nguerrera Sorry, the comment pre-dates our conversation on Teams yesterday. I could simply remove the references to `FileDefinition` and `FileDependency`.